content/en/blog/_posts/2022-07-11-meet-our-contributors-APAC-China-region-03.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-07-11T12:00:00+0000
+slug: meet-our-contributors-chn-ep-03
+canonicalUrl: https://www.kubernetes.dev/blog/2022/07/11/meet-our-contributors-chn-ep-03/
+---
+
+**Authors & Interviewers:** [Avinesh Tripathi](https://github.com/AvineshTripathi), [Debabrata Panigrahi](https://github.com/Debanitrkl), [Jayesh Srivastava](https://github.com/jayesh-srivastava), [Priyanka Saggu](https://github.com/Priyankasaggu11929/), [Purneswar Prasad](https://github.com/PurneswarPrasad), [Vedant Kakde](https://github.com/vedant-kakde)
+
+---
+
+Hello, everyone 👋
+
+Welcome back to the third episode of the "Meet Our Contributors" blog post series for APAC.
+
+This post features four outstanding contributors from  China, who have played diverse leadership and community roles in the upstream Kubernetes project.
+
+So, without further ado, let's get straight to the article.
+
+
+## [Andy Zhang](https://github.com/andyzhangx)
+
+Andy Zhang currently works for Microsoft China at the Shanghai site. His main focus is on Kubernetes storage drivers. Andy started contributing to Kubernetes about 5 years ago.
+
+He states that as he is working in Azure Kubernetes Service team and spends most of his time contributing to the Kubernetes community project. Now he is the main contributor of quite a lot Kubernetes subprojects such as Kubernetes cloud provider code.
```

Comment:
A little confused with `Kubernetes cloud provider code` if I localize this clause. Doe it refer to contributing code for Kubernetes cloud providers?

Reply:
Yeah cloud provider coder here means this repo https://github.com/kubernetes-sigs/cloud-provider-azure

Reply:
@windsonsea Any suggestions for this one or will the initial sentence work?

Reply:
@windsonsea Any suggestions for this one or will the initial sentence work?

Reply:
We'll use the wording from the upstream article (review feedback should go there).